contracts/libraries/Migrate_1_3_0_to_1_2_0.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+import "../common/SelfAuthorized.sol";
+
+/// @title Migration - migrates a Safe contract from 1.3.0 to 1.2.0
+/// @author Richard Meissner - <richard@gnosis.io>
+contract Migration is SelfAuthorized {
```

Comment:
This would be just for downgrading (emergency or testing purposes), right?

Reply:
yes, and as an example. I will make this more clear with https://github.com/gnosis/safe-contracts/issues/265 and https://github.com/gnosis/safe-contracts/issues/260